radar_control/radar_control.py

```diff
@@ -679,12 +679,19 @@ def radar():
 
                             if beam_time < aveperiod.intt*1e-3:
                                     #TODO maybe log skipping beam?
-                                msg = "Scan time too long, skipping beam {}"
-                                msg = msg.format(sm.COLOR("yellow", beam_iter))
-                                rad_ctrl_print(msg)
-                                continue
-                        integration_period_done_time = integration_period_start_time + \
+                                # msg = "Scan time too long, skipping beam {}"
+                                # msg = msg.format(sm.COLOR("yellow", beam_iter))
+                                # rad_ctrl_print(msg)
+                                # continue
+                                integration_period_done_time = integration_period_start_time + \
+                                            timedelta(milliseconds=beam_time * 1e3) #ms
+                            else:
+                                integration_period_done_time = integration_period_start_time + \
                                             timedelta(milliseconds=aveperiod.intt) #ms
+                        else:
```

Comment:
Just suggest adding a comment here since the if statement is large
```suggestion
                        else:  # no scanbound for this scan
```